### PR TITLE
Proper docs icon colors and sentence casing

### PR DESCRIPTION
**Pull request by llorca**
- Icons page icons are now using `$pt-icon-color` following #497 
- Various sentence casing type of things

### CODE REVIEW

File: packages/core/src/components/menu/_menu.scss [hunk 1]
```diff
@@ -24,16 +24,16 @@ JavaScript API
 The `Menu`, `MenuItem`, and `MenuDivider` components are available in the __@blueprintjs/core__
 package. Make sure to review the [general usage docs for JS components](#components.usage).
 
-The Menu API includes three stateless React components:
+The menu API includes three stateless React components:
```
Comment (giladgray) [inline packages/core/src/components/menu/_menu.scss]: i think this should remain capitalized... we're talking about our Menu API as a proper noun.

File: packages/docs/src/styleguide.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ npm install --save @blueprintjs/core react react-dom react-addons-css-transition
 Import components from the `@blueprintjs/core` module into your project.
 Don't forget to include the main CSS stylesheet too!
 
-**[See Components Usage for more complete installation instructions.](#components.usage)**
+[See **Components usage** for more complete installation instructions.](#components.usage)
```
Comment (giladgray) [inline packages/docs/src/styleguide.md]: it's more like `Components > Usage` (breadcrumbs) so capitalization makes sense for both words. i wouldn't be opposed to adding the `>` even.

File: packages/core/src/components/menu/_menu.scss (hunk 1 above)
Comment (llorca) [inline packages/core/src/components/menu/_menu.scss]: 👍  Choose between:
- menu
- `Menu`

File: packages/docs/src/styleguide.md (hunk 2 above)
Comment (llorca) [inline packages/docs/src/styleguide.md]: Fair, I'm gonna match what we do in all other files:
`Make sure to review the [general usage docs for JS components](#components.usage).`

File: packages/core/src/components/menu/_menu.scss (hunk 1 above)
Comment (giladgray) [inline packages/core/src/components/menu/_menu.scss]: the latter

**Pull request merged**